ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form.view.js

```diff
@@ -27,7 +27,12 @@ const Router = require('component/router/router')
 module.exports = Marionette.LayoutView.extend({
   template: template,
   tagName: CustomElements.register('search-form'),
-  className: 'is-button',
+  className() {
+    return this.model.get('createdBy') === 'system' &&
+      Router.attributes.path === 'forms(/)'
+      ? 'systemSearchForm'
```

Comment:
❓ 	Is this class used anywhere?

Reply:
this is to prevent the system search forms from acting like buttons on the search forms page